go/websocketbridge/main.go

```diff
@@ -0,0 +1,133 @@
+package websocketbridge
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/Originate/exocom/go/exorelay"
+	"github.com/Originate/exocom/go/structs"
+	uuid "github.com/satori/go.uuid"
+
+	"golang.org/x/net/websocket"
+)
+
+// Config contains the configuration values for ExoRelay
+type Config struct {
+	Host string
+	Port int
+	Role string
+}
+
+// WebsocketBridge handles communication between an Exosphere backend and
+// a browser front-end using websockets
+type WebsocketBridge struct {
+	ClientPort int
+	Config     Config
```

Comment:
I don't think `Config` or `ClientPort` needs to be saved on the Websocket bridge. `Config` was be passed into the `ConnectToExocom` method and `clientPort` can be passed into the `ListenForClients` methods.

It might also be a good idea to just create a single `Open` method which takes in both as I don't think we will want to one without the other